apps/server/src/shared/domain/entity/boardnode/boardnode.entity.ts

```diff
@@ -3,12 +3,11 @@ import { InternalServerErrorException } from '@nestjs/common';
 import { AnyBoardDo } from '../../domainobject';
 import { EntityId } from '../../types';
 import { BaseEntityWithTimestamps } from '../base.entity';
-import { BoardDoBuilder } from './types';
-import { BoardNodeType } from './types/board-node-type';
+import { BoardDoBuilder, BoardNodeType } from './types';
 
 const PATH_SEPARATOR = ',';
 
-@Entity({ tableName: 'boardnodes', discriminatorColumn: 'type' })
+@Entity({ tableName: 'boardnodes', discriminatorColumn: 'type', abstract: true })
```

Comment:
Is this already abstract? You have created a todo to use an abstract base class

Reply:
The todo that I created is for an abstract base class for root nodes like the ColumnBoardNode and the MediaBoardNode, but MikroORM has a bug with multiple abstract base classes so you can't do ColumnBoardNode -> RootBoardNode -> BoardNode atm. The @Entity({ abstract: true }) should have been there all along (if you can trust the MikroORM documentation).